.github/workflows/CD.yaml

```diff
@@ -133,6 +133,7 @@ jobs:
           RABBITMQ_PASS_HASH: ${{ secrets.RABBITMQ_PASS_HASH }}
           LOAD_GLEIF_GOLDEN_COPY: ${{ github.event.inputs.loadGleifGoldenCopy }}
           BACKEND_MIGRATION_BASELINE_VERSION: ${{ vars.BACKEND_MIGRATION_BASELINE_VERSION }}
+          COMMUNITY_MANAGER_MIGRATION_BASELINE_VERSION: ${{ vars.COMMUNITY_MANAGER_MIGRATION_BASELINE_VERSION }}
```

Comment:
why was this not necessary  before?

Reply:
Probably because migration just became relevant for the community manager in this story, correct @fsoffel?

Reply:
you are probably right